### ISSUE TITLE
Accessibility review

### ISSUE THREAD

**Issue by mark-prins** [msupply-foundation/open-msupply#9950]
## Is your feature request related to a problem? Please describe 👀
We are required to have an external accessibility review, evaluating the site against WCAG
A quick review of the dashboard shows 11 issues. Some of these can be easily addressed prior to an external review 

- [ ] Interactive element does not meet enhanced size (33 instances) **AAA**
_Interactive elements, like buttons or links, should be at least 44 by 44 pixels, so they can easily be clicked or tapped by all users_

- [ ] Button missing a text alternative (1 instance) **A**
_Buttons should always have a text label. The label can be inserted into the HTML — it does not need to be visible to pass this check_

- [ ] Visible label and accessible name do not match (1 instance) **A**
The accessible name of any interactive element should contain its visible text label.
